src/containers/home/home.component.jsx

```diff
@@ -80,16 +96,25 @@ const Home = ({
           } bg-blueGray w-full h-full `}
           onClick={() => sidebarViewThunk({ toggleView: false })}
         >
-          {postsFromState.map((post, index) => {
-            return (
-              <Post
-                post={post}
-                key={index}
-                user={userFromState}
-                fetchLatestPosts={fetchLatestPosts}
-              />
-            );
-          })}
+          <div
+            className={` ${
+              view === "userPosts" || view === "voteHistory" ? "hidden" : ""
+            } `}
+          >
+            <PostSort handleSort={handleSort} />
+          </div>
+          <div>
+            {postsFromState.map((post, index) => {
```

Comment:
Dont need index here can remove
```suggestion
            {postsFromState.map(post => {
```